src/pages/UserInfos/questions/Sexe.svelte

```diff
@@ -43,5 +43,10 @@
 <style>
 .sexe {
   display: flex;
+  font-weight: 500;
+}
+
+.sexe .form-check-input {
+  margin-right: 0.5rem !important
```

Comment:
Can you use a bootstrap `.ms-` class instead ?

Reply:
Need the !important. I use the px-2 on the other element to get the same result.

Reply:
`.ms-` already use `! important` 🤔 this class is a helper to avoid writing margin-left css code. I don't get your comment. Please use the bootstrap class.